### PR TITLE
Making Price History Component 

### PR DESCRIPTION
**Pull request by Kobe16**
In this branch: 
- Created price history component
- added bootstrap table for the component
- styled table to look like figma design
- fixed CSS on details page to make PriceHistory and CardRater components in-line

In the future: 
- will need to import data for price history table from a separate file

### CODE REVIEW

File: src/Components/PriceHistory/PriceHistory.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import React, { useState } from "react";
+import { Col, Table } from "react-bootstrap";
+import './PriceHistory.css';
+
+function PriceHistory() {
+
+    return (
+        <div className="PriceHistoryComponent">
```
Comment (jbrw1984) [inline src/Components/PriceHistory/PriceHistory.tsx:23]: This is a static component, I recommend figuring out how to pass in the array of data required to show the component and making it dynamic via props.

https://stackoverflow.com/questions/43948828/how-to-pass-an-array-of-items-in-react-js

File: src/Pages/DetailsPage/Details.tsx [hunk 2]
```diff
@@ -6,25 +6,19 @@ import TopNav from '../../Components/NavBar/Nav';
...
-            <ProductCard 
-              name={cardInfo.name}
-              image={cardInfo.image}
-              description={cardInfo.description}
-            />
-          ))}
-        </div> */}
-
+        <div className="InfoFlexBox">
+          <PriceHistory />
```
Comment (jbrw1984) [inline src/Pages/DetailsPage/Details.tsx]: You would pass in the props here.

**Review by jbrw1984 — DISMISSED** [review]
I think you could go ahead and setup the props and make the component dynamic.

**Comment by Kobe16** [comment]
I have now made the Price History Component dynamic by passing in the price history data as a prop. Currently, I only pass in the data for one specific pokemon card (the first card in the cardInfo array in cardInfo.ts). Also, I have not implemented a feature where the table is capped at 4 entries. Currently, it'll just render all different price history entries, even if there are 100 entries.

**Comment by averytribbett** [comment]
Looks good to me, I think only Reid can do the review though because he requested it.

**Review by averytribbett — APPROVED** [review]
I got it figured out, your changes look good.

**Pull request merged**